src/components/academy/game/create-initializer.js

```diff
@@ -7,14 +7,14 @@ export default function (StoryXMLPlayer, story, username, attemptedAll) {
 
   var hookHandlers = {
     startMission: function (number) {
-      document.getElementById('attempt').click()
+      history.push(`/academy/#{number}`)
```

Comment:
Is there a way to get the assessment type here? It needs it for the routing. If not, we can add a separate switch for this or remove the type in the routing entirely.

Reply:
Fixed with 8c262e7.